parser/src/consts/numbers.rs

```diff
@@ -0,0 +1,89 @@
+//! All the difrent enablable constants that can be used in Reti
+#![allow(clippy::excessive_precision)]
+
+///Mathematical constants
+pub mod math_constants {
+    pub use std::f64::consts::{E, LN_2, PI, SQRT_2, TAU};
+    /// The Euler-Mascheroni constant (γ)
+    pub const EGAMMA: f64 = 0.577215664901532860606512090082402431_f64;
+    /// The full circle constant (τ)
+    ///
+    /// Equal to 2π.
+    pub const PHI: f64 = 1.618033988749894848204586834365638118_f64;
+}
+///SI physics constants
+pub mod physics_si {
+    ///Universal physical constants
+    pub mod universal_constants {
+        /// Speed of light in m/s
+        pub const C: f64 = 299792458.0;
+        /// Plans constant
+        pub const H: f64 =
+            0.000_000_000_000_000_000_000_000_000_000_000_662_607_015;
```

Comment:
Use scientific notation instead, so for example instead of typing `let a = 0.000_000_000_123;` do `let b = 1.23e-10;`.

Reply:
did not know that rust suports that will use

Reply:
Yep it's nice, most languages have it. I think VS Code (via the rust LSP I guess) should have a feature so you can right click and convert to different number formats with one click instead of calculating it manually.

Reply:
good that i read this now after manualy verifying and counting all off them lol